ToolsTest/Models/VladyslavClassArticleTests/UpdateTextMethod_MSTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Tools.Models;
+
+namespace ToolsTest.Models.VladyslavClassArticleTests
+{
+    [TestClass]
+    public class UpdateTextMethod_MSTest
+    {
+        [TestMethod]
+        public void UpdateText_Text_NotNull()
+        {
+            //Arrange
+            Article _article = new Article();
+            string _text = "Hello";
+            //Act
+            _article.UpdateText(_text);
+            //Assert
+            Assert.IsNotNull(_article);
```

Comment:
The `UpdateText()` method changes the `Text` property of the `_article`, so need to be tested exactly that one
```suggestion
            Assert.IsNotNull(_article.Text);
```